File: clinic-core/src/main/java/com/nonier/cliniccore/entity/Message.java
package com.nonier.cliniccore.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.CreatedDate;

import java.time.LocalDateTime;

@Data
@Entity
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "message")
public class Message    {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "dialog_id", referencedColumnName = "id")
    private Dialog dialog;

    @Column(name = "text")
    private String text;

    @CreatedDate
    @Column(name = "creation_date")
    private LocalDateTime creationDate;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_from", referencedColumnName = "id")
    private User userFrom;

}

File: clinic-frontend/src/app/model/Message.ts
import {User} from "./User";

export class Message {

  id: number;
  text: string;
  dialogId: number;
  creationDate: Date;
  userFrom: User;

  constructor(id: number, text: string, dialogId: number, creationDate: Date, userFrom: User) {
    this.id = id;
    this.text = text;
    this.creationDate = creationDate;
    this.userFrom = userFrom;
  }
}
